### ISSUE TITLE
Works on Windows?

### ISSUE THREAD

**Issue by tbassetto** [rexxars/sanipack#1]
Hei!

Not sure why but it always fails on the `.editorconfig` step on my machine, not sure if it's because I run Windows, an old (but still supported) version of Node.js:
```
npx sanipack init sanity-plugin-testing-sanipack

[...]

[info] Wrote dist config (config.dist.json)
[info] Wrote package.json
[info] Wrote license file (LICENSE)
[info] Wrote readme file (README.md)
[error] ENOENT: no such file or directory, open 'C:\Users\[...]\AppData\Roaming\npm-cache\_npx\32148\node_modules\sanipack\src\assets\splat\.editorconfig'
```

* node v10.23.0
* npm 6.14.11

**Comment by dorelljames** [comment]
I'm seeing the same issue but here on my Mac.

![image](https://user-images.githubusercontent.com/977413/107211466-8bc00300-6a40-11eb-8a09-5e3c3cc74650.png)

- node v12.16.3
- npm 6.14.4

**Comment by dorelljames** [comment]
Just an update:

I don't know how to say this but I cloned the repo and use it by global package linking `npm-link` and it worked.

**Comment by rexxars** [comment]
Thanks for reporting! I had an `.npmignore` deeply inside the module which was meant to be copied to bootstrapped projects, but it was ironically ignoring files (`.editorconfig` and such) from being included in the published release! Awkward!

Released a new version, v1.0.6 which should fix it! Thanks again!
